src/main/java/com/puppycrawl/tools/checkstyle/api/FileContents.java

```diff
@@ -328,9 +329,11 @@ private boolean hasIntersectionWithSingleLineComment(int startLineNo, int startC
      * Checks if the current file is a package-info.java file.
      *
      * @return true if the package file.
+     * @deprecated use {@link CheckUtil#isPackageInfo(String)}
+     *              and {@link AbstractCheck#getFilePath()} instead.
      */
+    @Deprecated(since = "10.2")
```

Comment:
Do we have some check that will fail build if deprecated method is used?

Reply:
yes, there was some check, but I cannot say which one

Reply:
Resolved.